COPY/etc/httpd/conf.d/manageiq-balancer-ui.conf

```diff
@@ -10,3 +10,16 @@ BalancerMember http://0.0.0.0:3007
 BalancerMember http://0.0.0.0:3008
 BalancerMember http://0.0.0.0:3009
 </Proxy>
+
+<Proxy balancer://evmcluster_ui_ws/ lbmethod=bybusyness>
```

Comment:
Apache does not support protocol independent proxying.

So we have to setup a second proxy.
It looks like a completely independent set of servers to apache, but this is the best we can do

Reply:
If you note, this looks very similar to the websocket proxy over in [manageiq-balancer-websocket.conf](https://github.com/ManageIQ/manageiq-appliance/blob/dd21a5cd4a84be210d69e47391149a4211c27197/COPY/etc/httpd/conf.d/manageiq-balancer-websocket.conf)

Reply:
Very similar = exactly the same :wink:

Reply:
The re-use of the 3000-3009 ports on different protocols, while correct, could be confusing if we ever log by port or us system tools (netstat/etc.) and forget to filter by protocol... we could have more listening things on port 3000 than we expected.

Does it make sense to use a unique different port range?

Reply:
@jrafanie HTTP and WebSocket are the same protocol just used differently, we don't have two services listening on 300x here, but a rack server using two different middlewares determined by URL.

Reply:
The problem is with apache's configuration, it needs to have the `http` and `ws` prefix to make this work :disappointed:

Reply:
@skateman Should the `disablereuse=on` also be on the other evmcluster_ui ones?

Reply:
Oh nvm...I read the docs for that setting and it doesn't do what I thought it did...I thought it was closer to something like `SO_REUSEADDR`